apps/emqx_rule_engine/src/emqx_rule_engine_jwt_worker.erl

```diff
@@ -0,0 +1,220 @@
+%%--------------------------------------------------------------------
+%% Copyright (c) 2022 EMQ Technologies Co., Ltd. All Rights Reserved.
+%%
+%% Licensed under the Apache License, Version 2.0 (the "License");
+%% you may not use this file except in compliance with the License.
+%% You may obtain a copy of the License at
+%%
+%%     http://www.apache.org/licenses/LICENSE-2.0
+%%
+%% Unless required by applicable law or agreed to in writing, software
+%% distributed under the License is distributed on an "AS IS" BASIS,
+%% WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+%% See the License for the specific language governing permissions and
+%% limitations under the License.
+%%--------------------------------------------------------------------
+
+-module(emqx_rule_engine_jwt_worker).
+
+-behaviour(gen_server).
+
+%% API
+-export([ start_link/2
+        , lookup_jwt/1
+        , lookup_jwt/2
```

Comment:
we need a module `emqx_rule_engine_jwt.erl` to host the APIs (and the ETS table since it's shared by workers).

Reply:
:heavy_check_mark: 
https://github.com/emqx/emqx/pull/9241/commits/ec56a9f842a7fefa90df259d75ae9dde63cf0451